javascript/javascript1/week3/note.js

```diff
@@ -0,0 +1,65 @@
+const notes = [];
+
+// implementing SAVE note
+function saveNote(content, id) {
+    const noteObject = { content: content, id: id };
+    notes.push(noteObject);
```

Comment:
An id is by deifinition something used to uniquely identify an item. Consider what happens when we add another item with the same '1'. How will this affect your code.

Reply:
Added id check for being unique